### ISSUE TITLE
Actor Set Relative Position

### ISSUE THREAD

**Issue by josefnpat** [chrismaltby/gb-studio#38]
I see there's Actor Set Position, but there is no Actor Set Relative Position!

**Comment by ISSOtm** [comment]
What does that mean?

**Comment by chrismaltby** [comment]
It’s okay I know what josefnpat means :-) there’s currently scripting events available for setting the x/y tile position of an actor to absolute values but no way to say move an actor two tiles to the left. Shouldn’t be too much of a problem and probably should add Actor Move Relative at the same time.

**Comment by thomas-alrek** [comment]
If my feature #76 is accepted, you could also use that to emulate this behaviour.

E.g, "Move Relative" x: 1, y: 0

```
Actor Get Position:
    actor_id,
    variable_x, variable_y

Increment Variable By 1:
    variable_x

Actor Set Position To Value:
    actor_id,
    variable_x, variable_y
```
This would result in the actor with id "actor_id" moving 1 unit to the right relative to its current position

**Comment by thomas-alrek** [comment]
This issue is fixed in commit 6c53c7c and can be closed.
@josefnpat
@chrismaltby
